src/TinkerServiceProvider.php

```diff
@@ -18,9 +18,9 @@ class TinkerServiceProvider extends ServiceProvider implements DeferrableProvide
     public function boot()
     {
         $source = realpath($raw = __DIR__.'/../config/tinker.php') ?: $raw;
-
+        
         if ($this->app instanceof LaravelApplication && $this->app->runningInConsole()) {
-            $this->publishes([$source => config_path('tinker.php')]);
+            $this->publishes([$source => $this->app->configPath('tinker.php')]);
```

Comment:
interface available in Laravel Framework 6: https://github.com/laravel/framework/blob/10.x/src/Illuminate/Contracts/Foundation/Application.php#L88-L93